src/main/java/org/spongepowered/api/data/manipulator/immutable/tileentity/ImmutableFurnaceData.java

```diff
@@ -28,46 +28,48 @@
 import org.spongepowered.api.data.manipulator.ImmutableDataManipulator;
 import org.spongepowered.api.data.manipulator.mutable.tileentity.FurnaceData;
 import org.spongepowered.api.data.value.immutable.ImmutableBoundedValue;
-import org.spongepowered.api.entity.Item;
 
 /**
- * An {@link ImmutableDataManipulator} representing the {@link Furnace}
- * information such as the {@link #remainingBurnTime()} and
- * {@link #remainingCookTime()}.
+ * An {@link ImmutableDataManipulator} representing the {@link Furnace}.
  */
 public interface ImmutableFurnaceData extends ImmutableDataManipulator<ImmutableFurnaceData, FurnaceData> {
 
     /**
-     * Gets the {@link ImmutableBoundedValue} for the remaining burn time of the
-     * {@link Furnace}.
+     * Gets the {@link ImmutableBoundedValue} for the already passed burn time of the {@link
+     * Furnace}. When the {@link #passedBurnTime()} is equal to the {@link #maxBurnTime()}, the
+     * current used fuel is depleted.
      *
-     * @return The immutable value for the remaining burn time
+     * @return The immutable value for the already passed burn time
      */
-    ImmutableBoundedValue<Integer> remainingBurnTime();
+    ImmutableBoundedValue<Integer> passedBurnTime();
 
     /**
-     * Gets the {@link ImmutableBoundedValue} for the remaining cook time of the
-     * {@link Furnace}.
+     * Gets the {@link ImmutableBoundedValue} for the maximum amount of fuel that can be supplied
+     * with the used fuel item.
      *
-     * <p>The {@link #remainingCookTime()} is the difference of {@link #maxCookTime()} and
-     * the time the item has already been cooked.</p>
+     * It is represented by the flame icon in the {@link Furnace}, if the flame is 100% filled the
+     * value is exact this one. So its the maximum of the {@link #passedBurnTime()}.
```

Comment:
if the flame is 100% filled the value is exact**ly** this one, this makes it the maximum of the {@link #passedBurnTime()}.

Still a bit wonky (my try too).